File: samples/sample.cpp
#include <iostream>

using namespace std;

int nth_fibonacci(int n) {
    if (n <= 1)
        return n;
    return nth_fibonacci(n - 1) + nth_fibonacci(n - 2);
}

int main() {
    int n = 40;
    int result = nth_fibonacci(n);
    
    cout << "testing from C++" << endl;
    
    return 0;
}

File: samples/sample.java
public class sample {
    static int nthFibonacci(int n) {
        if (n <= 1) {
            return n;
        }
        return nthFibonacci(n - 1) + nthFibonacci(n - 2);
    }

    public static void main(String[] args) {
        int n = 40;  // Número de Fibonacci a calcular
        nthFibonacci(n);
        System.out.print("testing from java");
    }
}